kymatio/scattering2d/backend/torch_backend.py

```diff
@@ -0,0 +1,328 @@
+# Authors: Edouard Oyallon, Sergey Zagoruyko
+
+import torch
+from torch.nn import ReflectionPad2d
+from collections import namedtuple
+
+BACKEND_NAME = 'torch'
+
+def iscomplex(x):
```

Comment:
Do we want doc strings for this? Is the method name fine?

Reply:
no.

Reply:
Agreed. Prefix with _

Reply:
Done (here and in `torch_skcuda_backend`).